docs/content/reference/resource-schema/azure/index.md

```diff
@@ -0,0 +1,8 @@
+---
+type: docs
+title: "Supported Azure resources"
+linkTitle: "Supported Azure resources "
+manualLink: "https://learn.microsoft.com/en-us/azure/templates/"
```

Comment:
For Azure links we should remove the `/en-us` portion for URLs.

https://learn.microsoft.com/en-us/azure/templates/ will always direct to the English version of the docs.

https://learn.microsoft.com/azure/templates/ will direct to the localized docs for the user's default language setting.